source/_integrations/motion_blinds.markdown

```diff
@@ -25,7 +25,15 @@ Additionally the following brands have been reported to also work with this inte
 - [Dooya](http://www.dooya.com/)
 - [Bloc Blinds](https://www.blocblinds.com/)
 - [Brel Home](https://www.brel-home.nl/)
-- [Smart Home](https://www.smart-home.hu) 
+- [Smart Home](https://www.smart-home.hu)
+- [Hurrican Shutters Wholesale](https://www.hurricaneshutterswholesale.com/)
+- [iSmartWindow](https://www.ismartwindow.co.nz/)
+- [Raven Rock MRG](https://www.ravenrockmfg.com/)
+- [AMP Motorization](https://www.ampmotorization.com/)
+- [Uprise Smart Shades](http://uprisesmartshades.com)
+- [Bliss Automation - Alta Window Fashions](https://www.altawindowfashions.com/product/automation/bliss-automation/)
+- [3 Day Blinds](https://www.3dayblinds.com/)
+- [Martec](https://www.martec.co.nz/)
```

Comment:
Let's short this list... 🔠  🤘

Reply:
I sorted the bottom part alphabetically.
I wanted to keep Motion Blinds, Smart Blinds and Dooya on top since this integration is actually written using the API documentation of those two and I am in direct contact with those companies. So they are the main focus of this integration.

The rest is just reported to work but not tested by me.

Reply:
Never mind, this does not make much sense.
I now just sorted alphabetically.

On top it is already written that this is developed for Motion Blinds.